Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs

```diff
@@ -22,5 +22,14 @@ public static void TaskTurnPedToFaceEntity(Ped ped, Entity entity, int duration)
         {
             NativeFunction.Natives.TASK_TURN_PED_TO_FACE_ENTITY(ped, entity, duration);
         }
+
+        /// <summary>
+        /// Causes the given ped to shoot at the specified target ped.
+        /// </summary>
+        /// <param name="unknown">Unknown parameter, default 0</param>
+        public static void TaskVehicleShootAtPed(Ped ped, Ped target, float unknown = 0)
```

Comment:
The name of this method confuses me. A vehicle should shoot at a ped?

Reply:
It follows the naming convention as described in the readme.  The native's name is TASK_VEHICLE_SHOOT_AT_PED.

Reply:
We should discuss this one. It's very confusing when it does not make use of any vehicle. The provided wrappers should be helpful for other developers, making Rage easier too use and give the code a better readability.

Reply:
If the standard is to match the language GTA/Natives use, I do not want to deviate.  When people search for how to get a ped to shoot from a vehicle, they will come across the TASK_VEHICLE_SHOOT_AT_PED native.  I understand the name is not as clear as you would like, but I also provided an extension method with a more user friendly name to go along with this.

Reply:
""When people search for how to get a ped to shoot from a vehicle"" - so this native is actually for a ped who should shoot while it is inside a vehicle?

Reply:
Yes, as the summary says.

Reply:
As discussed, this Native does only work with vehicles. Please extend the summary to make this more clear for other developers.